server/sonar-db-dao/src/main/java/org/sonar/db/component/ComponentQuery.java

```diff
@@ -48,6 +49,7 @@ public ComponentQuery(@Nullable String nameOrKeyQuery, String... qualifiers) {
     this.language = null;
     this.componentIds = null;
     this.isPrivate = null;
+    this.lastAnalysisBefore = null;
```

Comment:
I would rename it to `analyzedBefore` => it's a project search, it doesn't make sense to search for project having been analyzed in the past : I would not put the "last" word